### DISCUSSION TITLE
ARCitect 0.0.36 Add Assay Dialogue missing the option to reference a study

### DISCUSSION THREAD

**Discussion by ZimmerD** [nfdi4plants/ARCitect#207] [General]
Previously it was possible to assign an assay to a study within the creation dialogue. (see https://www.youtube.com/watch?v=_9vEitxLsv4&list=PLi6d1LSow4XBZ4MKMMGOW6qDTZwe7Zg8w&index=9&ab_channel=NFDI4plants at 0:20). In the current version this feature is missing. Where should I now reference the study?

**Comment by Freymaurer** [comment]
ARCitect optet for the lowest complexity solution. Explicit assay referencing in studies is optional per ARC specification and to reduce merge conflicts we decided to not do it with the ARCitect tool. I hope this answers your question. Feel free to give feedback :)

**Reply by ZimmerD** [reply]
Ah okay, this makes sense! I have to say that I personally, liked the option to specify the study right away (in my ARCs 99% of assays belong to an study), but that is maybe just personal preference. Maybe @Brilator can add his thoughts on this? 

If we go with the current design I would have to update the video, for this I would need to know whats the intended way to specify an study an assay belongs to.

**Reply by Freymaurer** [reply]
You can if you want edit the xlsx file, but as it is not required we currently support the least freaction solution of not referencing the assays.

Relation will also be inferrable due to references of input/output

**Reply by Brilator** [reply]
> Relation will also be inferrable due to references of input/output

Yes! I think that's more important than the "registration" 

We'll just need to make this connection more visible / visualizable / automagically visualized on commit.

**Reply by Freymaurer** [reply]
Yes! Visualisation is a huge topic we are investigating. We also think that the overview gained through registration is very minimal. As you still need to click into multiple files and it is not really well formatted for this kind of information.

Current ideas are:
- Mermaid graphs in README.md
- cytoscape visualisation tool
